xlgui/widgets/playlist.py

```diff
@@ -236,7 +236,7 @@ def _get_pl_func(o, c):
     items.append(menuitems.ExportPlaylistMenuItem('export', ['tab-close-sep'], _get_pl_func))
     items.append(menuitems.ExportPlaylistFilesMenuItem('export-files', ['export'], _get_pl_func))
     items.append(sep('tab-export-sep', ['export']))
-    items.append(smi('tab-close', ['tab-export-sep'], None, 'gtk-close',
+    items.append(smi('tab-close', ['tab-export-sep'], _("Close _Tab"), 'window-close',
```

Comment:
I've changed it the other way: `_Close Tab` → `Close _Tab` since I need `_C` for "Clear"